parachain/runtime/kintsugi/src/dex.rs

```diff
@@ -74,6 +75,7 @@ impl dex_swap_router::Config for Runtime {
     type CurrencyId = CurrencyId;
     type NormalAmm = DexGeneral;
     type StableAMM = DexStable;
+    type MaxSwaps = ConstU16<4>;
```

Comment:
Where did you come up with this value? Are you sure that will be sufficient?

Reply:
I believe that was the limit that was also used by the UI for the route calculation

Reply:
We can set it higher if you think it's necessary but with our planned pools I don't think it's needed

Reply:
You are right the hop limit is 4 I believe